services/order.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib.auth.models import User
+from django.db import transaction
+
+from db.models import Order
+
+
+def create_order(tickets, username, date=None):
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order
```

Comment:
The `create_order` function is incomplete. You need to implement the logic to create an `Order` instance and associate the provided tickets with this order. Ensure you save the `Order` instance and update the `tickets` to reference this order.